styles/styles.css

```diff
@@ -534,3 +534,106 @@ img[src="/icons/loading.svg"] {
   height: 124px;
   filter: invert(1);
 }
+
+/* MARK: Author list */
+ul.author-list {
+  list-style: none;
+  margin: 20px 0;
+  padding: 0;
+}
+
+ul.author-list .author {
+  margin-bottom: 10px;
+  width: 100%;
+  display: flex;
+  gap: 6px;
+}
+
+ul.author-list:empty::after {
+  content: "No external authors yet";
+}
+
+ul.author-list .author .revoke-button {
+  padding: 0;
+  width: 28px;
+  height: 28px;
+  border: none;
+  background: url("/icons/remove-circle.svg") no-repeat center / contain;
+}
+
+.add-author-section {
+  border-top: 1px solid var(--gray);
+  margin-bottom: 24px;
+}
+
+.add-author-section h4 {
+  margin: 6px 0;
+}
+
+.add-author-section form {
+  display: flex;
+  gap: 10px;
+  margin-bottom: 8px;
+}
+
+/* MARK: Help Button */
+.help-btn {
+  position: fixed;
+  right: 0;
+  bottom: 0;
+  margin: 1rem;
+  font-size: 1.8rem;
+  z-index: 999;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 58px;
+  height: 58px;
+  border: 1px solid var(--white);
+  border-radius: 50%;
+  background-color: var(--orange);
+  color: white;
+  text-align: center;
+  text-decoration: none;
+  font-weight: 600;
+  cursor: pointer;
+  box-shadow: 0 0 5px var(--box-shadow-color);
+}
+
+.help-btn:hover {
+  transform: scale(1.1);
+}
+
+/* MARK: Expedition.js */
+.expedition-popover {
+  min-width: 320px !important;
+  max-width: 400px;
+}
+
+.expedition-popover:not(:has(.expedition-popover-arrow)) {
+  margin-left: -60px;
+}
+
+
+#expedition-popover-title, #expedition-popover-footer {
+  background-color: transparent;
+  color: black;
+  height: auto;
+}
+
+.expedition-popover-btn, .expedition-popover-prev-btn, .expedition-popover-next-btn, .disable-tour-btn {
```

Comment:
Would it be possible for consistency to reuse the button styles we already use across kestrelone ?